app/src/main/java/org/greenstand/android/TreeTracker/database/entity/PlanterInfoEntity.kt

```diff
@@ -40,7 +40,7 @@ data class PlanterInfoEntity(
         const val TABLE = "planter_info"
 
         const val ID = "_id"
-        const val IDENTIFIER = "planterInfoId"
+        const val IDENTIFIER = "planter_identifier"
```

Comment:
This change is already made in 1.3. I find `planterInfoId` confusing and changed it to `planter_identifier` (this way it is distinct from `_id` that belongs to `planter_info` table).  Also using snake case for the column name.